chains/corda/codec.go

```diff
@@ -3,12 +3,14 @@ package corda
 import (
 	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
 	"github.com/cosmos/cosmos-sdk/simapp/params"
+	cordatypes "github.com/datachainlab/corda-ibc/go/x/ibc/light-clients/xx-corda/types"
 	"github.com/datachainlab/relayer/core"
 )
 
 // RegisterInterfaces register the ibc transfer module interfaces to protobuf
 // Any.
 func RegisterInterfaces(registry codectypes.InterfaceRegistry) {
+	cordatypes.RegisterInterfaces(registry)
```

Comment:
You might want to implement AppModuleBasic in corda-ibc and add it to the following location:
https://github.com/datachainlab/relayer/blob/305ed1ca8decfda340b53c77d6c36f76a11e09ef/core/encoding.go#L48